integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -54,8 +54,8 @@ public static ArgumentParser parse(String clientUrl) {
         ArgumentParser.builder()
             .host(uri.getScheme() + "://" + uri.getAuthority())
             .timeout(getTimeout(nameValuePairList))
-            .apiKey(getNamedStringParameter(nameValuePairList, "api_key"))
-            .overwriteName(getNamedStringParameter(nameValuePairList, "overwrite_name"))
+            .apiKey(getNamedStringParameter(nameValuePairList, "api-key"))
```

Comment:
Why change this parameter's name? It could break existing code.

Reply:
apparently there are best practises for naming url parameters and use of hyphens is recommended, but yeah backward compatibility is more important, I'll revert that